package.json

```diff
@@ -0,0 +1,149 @@
+{
+  "name": "terra-graphs",
+  "private": true,
+  "version": "0.1.0",
+  "description": "terra-graphs",
+  "engines": {
+    "node": ">=8.9.2 <12"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-graphs.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Terra Graphs",
+    "terra-graphs"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-graphs/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "@cerner/terra",
+    "overrides": [
+      {
+        "files": [
+          "**/*/terra-dev-site/**/*.jsx"
+        ],
+        "rules": {
+          "import/no-extraneous-dependencies": "off",
+          "import/no-unresolved": "off",
+          "import/extensions": "off"
+        }
+      },
+      {
+        "files": [
+          "*.doc.jsx"
+        ],
+        "rules": {
+          "import/no-webpack-loader-syntax": "off",
+          "import/first": "off",
+          "import/no-duplicates": "off"
+        }
+      }
+    ],
+    "rules": {
+      "react/forbid-prop-types": "off"
+    },
+    "parser": "babel-eslint",
+    "settings": {
+      "polyfills": [
+        "Promise",
+        "Map"
+      ]
+    }
+  },
+  "stylelint": {
+    "extends": "stylelint-config-terra",
+    "rules": {
+      "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
+    }
+  },
+  "scripts": {
+    "bootstrap:hoist": "lerna bootstrap --hoist",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
+    "clean:install": "npm run clean && npm install",
+    "clean:obsolete-snapshots": "npm test -- -u",
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:build": "lerna ls | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
```

Comment:
technically we can use lerna to run this in parallel without this weirdnes
```
lerna run compile --parallel
```

Reply:
yes but it would be slower since we'd run babel X times verses running it once with x directories.